fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/BrowserRobot.kt

```diff
@@ -915,8 +915,21 @@ class BrowserRobot {
     fun fillPdfForm(name: String) {
         // Set PDF form text for the text box
         itemWithResId("pdfjs_internal_id_10R").setText(name)
-        // Click PDF form check box
-        itemWithResId("pdfjs_internal_id_11R").click()
+        mDevice.waitForWindowUpdate(packageName, waitingTime)
+        if (
+            itemWithResId("pdfjs_internal_id_11R").exists().not() &&
```

Comment:
itemWithResId("pdfjs_internal_id_11R").exists().not() can be !itemWithResId("pdfjs_internal_id_11R").exists()